layers/+tools/shell/packages.el

```diff
@@ -163,10 +163,21 @@
     (progn
       (add-to-list 'term-bind-key-alist '("<tab>" . term-send-tab))
       ;; multi-term commands to create terminals and move through them.
-      (spacemacs/set-leader-keys-for-major-mode 'term-mode
-        "c" 'multi-term
-        "p" 'multi-term-prev
-        "n" 'multi-term-next))))
+      (if (member dotspacemacs-editing-style '(vim hybrid))
+          (progn
+            (spacemacs/set-leader-keys-for-major-mode 'term-mode
+              "c" 'multi-term
+              "N" 'multi-term-prev
```

Comment:
No need to split this based on editing style, you can have both `N` and `p` bound to `multi-term-prev` for all 3.